org-roam.el

```diff
@@ -1152,45 +1152,54 @@ Applies `org-roam-link-face' if PATH corresponds to a Roam file."
 
 ;;;; org-roam-backlinks-mode
 (defvar org-roam-backlinks-mode-map
-  (let ((km (make-sparse-keymap)))
-    (define-key km [mouse-1] 'org-roam-open-at-point)
-    (define-key km (kbd "RET") 'org-roam-open-at-point)
-    km)
+  (let ((map (make-sparse-keymap)))
+    (define-key map [mouse-1] 'org-open-at-point)
+    (define-key map (kbd "RET") 'org-open-at-point)
+    map)
   "Keymap for `org-roam-backlinks-mode'.")
 
-(define-derived-mode org-roam-backlinks-mode org-mode "Backlinks"
-  "Major mode for the `org-roam-buffer'.
+(define-minor-mode org-roam-backlinks-mode
+  "Minor mode for the `org-roam-buffer'.
 
-\\{org-roam-backlinks-mode-map}")
+\\{org-roam-backlinks-mode-map}"
+  :lighter " Backlinks"
+  :keymap org-roam-backlinks-mode-map
+
+  (if org-roam-backlinks-mode
+      (add-hook 'org-open-at-point-functions
+                'org-roam-open-at-point nil 'local)
+    (remove-hook 'org-open-at-point-functions
+                 'org-roam-open-at-point 'local)))
 
 (defun org-roam-open-at-point ()
```

Comment:
Note that this function now only does the Org-roam part and as a consequence is no longer interactive (in order not to confuse people). Would you prefer to keep an interactive `org-roam-open-at-point` with the previous behavior (to fall back to `org-open-at-point`)? Or is it okay if we direct users to change their bindings/habits to directly make use of `org-open-at-point`?